websocket/backend_server/README.md

```diff
@@ -0,0 +1,29 @@
+## Ballerina with Websockets
```

Comment:
Why using level 2 heading here. It should be L1.

Reply:
updated.